README.md

```diff
@@ -87,3 +87,28 @@ The following directions use Intellij as the IDE, but the steps should be simila
     ```
 ### Debugging / Calling end points 
   * Once you're running the project, you can double check if all is well by going to ``` localhost:8081 ``` for the admin interface (if ```0.0.0.0:8081``` isn't working for you)
+
+### Zerobase Backend PDF / QR Generation
+------------------------------
+Libraries: zxing, thymeleaf, itext, flying saucer
+
+PDF and QR generation files exist here: 
+```
+src/main/kotlin/io/zerobase/smarttracing/resources/PdfGenerator.kt
+src/main/kotlin/io/zerobase/smarttracing/resources/QRCodeGenerator.kt
+```
+
+PDF Configurations (via html, css, img, asset templates etc) here: 
+```
+src/main/resources/pdfconfig
+```
+
+To generate a PDF for yourself, you can run the main function in GeneratePdf (```PdfGenerator.kt```). 
+    
+A copy of the generated PDF is checked-in, in case you want to see what is generated:
+
+    src/main/resources/pdfconfig/example-zerobase-qr.pdf        
```

Comment:
everything in the `src/main/resources` folder will be bundled with the jar when we build. if it's not needed at runtime, don't put it here. examples are good, just put them somewhere like an `example-pdfs`  folder in the root of the project